src/hooks/useActiveContactGroup.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { useActiveItem } from './useActiveItem';
+import { ContactGroup } from '../model/contact-group';
+import { useContactGroupStore } from '../store/contact-groups';
+
+export const useActiveContactGroup = (): ContactGroup | undefined => {
+	const contactGroups = useContactGroupStore((state) => state.storedContactGroups);
+
+	const { activeItem } = useActiveItem();
+
+	return contactGroups.find((contactGroup) => contactGroup.id === activeItem);
```

Comment:
This active works only for the already loaded items. In case of refresh with the displayer opened on an item of a page which is not the first one, the displayer is not shown, but in the url the id is visible.
Track down this issue somewhere, it is not a blocker, but I think the displayer should fall back to a network request in which only the activeId data are requested.

Reply:
It is moves to IRIS-4994